### PR TITLE
Feature/typos

### PR DESCRIPTION
**Pull request by mdavy86**
Two small bugs, setup.R fixed, `data/` will always be false

```
## Assuming data exists
file.exists("data/")
[1] FALSE
 file.exists("data")
[1] TRUE
```

dplyr version 0.4.0 introduces `right_join(x, y)`, and `full_join(x, y)`, I've added a conditional to check the package version installed, this code is optional asuming people are going to install >=R-3.1.3 and the latest version of `dplyr`
- http://blog.rstudio.org/2015/01/09/dplyr-0-4-0/
- http://stackoverflow.com/questions/21841146/is-there-an-r-dplyr-method-for-merge-with-all-true

### CODE REVIEW

File: 01-intro-to-dplyr.Rmd [hunk 1]
```diff
@@ -136,7 +136,7 @@ to describe how to break a dataset down into groups of rows.
 Let's calculate mean weight and hindfoot length for each species:
 
 ```{r group-by-example, results='show', purl=FALSE}
-by_species <- group_by(surveys, species_id)
+by_species    <- group_by(surveys, species_id)
```
Comment (ethanwhite) [inline 01-intro-to-dplyr.Rmd]: Why the extra spaces?

**Comment by ethanwhite** [comment]
Overall this looks good to me. The one thing I don't like is the addition of the explicit checks to see if the version is sufficient to run `right_join` and `full_join`. I think that should be handled by clearly specifying the required version, not by doing an explicit check. Since the version requirement has already been noted above I think we could just leave that code the way it was.

**Comment by fmichonneau** [comment]
Hi Marcus,

Thanks for these fixes!

I agree with Ethan:
- could you please remove the extra spaces?
- The lessons are mostly intended to be rendered by instructors, and rendering shouldn't be conditional on the versions of the package. It is assumed that the latest versions of R/dplyr will be used for this. We don't want to maintain backward compatibility with older versions.

**Comment by mdavy86** [comment]
Hi, sure I can remove the spaces, the reason was readibility lining up on the assign operator, but not important here.

``` r
by_species    <- group_by(surveys, species_id)
species_stats <- summarise(by_species,
  count = n())
```

I agree with the removal of the dplyr conditional, just wanted to highlight as knitr will fail on that markdown document (which happened with R-3.1.1 for me, but works on R-3.1.3), will add changes to the pull request...

**Comment by mdavy86** [comment]
Pull request updated in commit [727746b](https://github.com/mdavy86/R-dplyr-ecology/commit/727746ba6b7271016781833ec1661a611908c73d)

File: setup.R [hunk 2]
```diff
@@ -1,6 +1,8 @@
+#!/bin/env Rscript
```
Comment (fmichonneau) [inline setup.R]: We don't need this as it is not meant to be executed as a standalone script, just embedded within the Rmd file.

File: 02-data-manipulation.Rmd [hunk 3]
```diff
@@ -36,8 +36,8 @@ In particular for larger datasets, it can be tricky to remember the column
 number that corresponds to a particular variable. (is hindfoot length in column 7
 or 9? oh, right... they are in column 8). In some cases, in which column the
 variable will be can change if the script you are using adds or removes
-columns. It's therefore often better to use column names to refer to a
-particular variable, and it makes your code easier to read and your intentions
+columns. It is usually recommended use column names to refer to a
```
Comment (fmichonneau) [inline 02-data-manipulation.Rmd]: recommended to

File: 02-data-manipulation.Rmd [hunk 4]
```diff
@@ -36,8 +36,8 @@ In particular for larger datasets, it can be tricky to remember the column
...
+particular variable, it makes your code easier to read, and your intentions
```
Comment (fmichonneau) [inline 02-data-manipulation.Rmd]: , as it makes your code easier to read,

File: setup.R (hunk 2 above)
Comment (mdavy86) [inline setup.R]: Habit when writing *.R scripts. An alternative is something like

```
#!/bin/echo Source me don't execute
```

if a user executes from the shell without specifying explicitly to use `Rscript`

I will remove the shebang.

File: 02-data-manipulation.Rmd (hunk 4 above)
Comment (mdavy86) [inline 02-data-manipulation.Rmd]: think I've fixed it now

**Comment by fmichonneau** [comment]
thanks @mdavy86!

**Pull request merged**